src/styles/calendar.scss

```diff
@@ -0,0 +1,41 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  padding-top: $calendar-top-padding;
+  gap: $day-gap;
+
+  &__day {
+    @include square ($day-size);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid black;
+    background-color: $day-color;
+    transition: 0.5s all;
```

Comment:
good job! I did not find any mistakes, but I think it would be better to add a transition only for color and transform properties, instead of all properties